src/Compilers/VisualBasic/Portable/VisualBasicParseOptions.vb

```diff
@@ -19,6 +19,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Private _features As ImmutableDictionary(Of String, String)
 
         Private _preprocessorSymbols As ImmutableArray(Of KeyValuePair(Of String, Object))
+        Private _specifiedLanguageVersion As LanguageVersion
         Private _languageVersion As LanguageVersion
```

Comment:
True, but not always from a constructor. I do not intend to refactor the way the fields are maintained for this PR.

---
In reply to: [74520116](https://github.com/dotnet/roslyn/pull/12969#discussion_r74520116) [](ancestors = 74520116)